tox.ini

```diff
@@ -0,0 +1,22 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py33, py34, pypy
+
+#, flake8
+
+[testenv]
+deps = 
+	-rrequirements.txt
+	-rtest-requirements.txt
+commands = nosetests -v --logging-level=DEBUG
+
+[testenv:py34]
+commands = nosetests -v --logging-level=DEBUG --cov streamparse --with-cov
```

Comment:
The only reason this test command is different in `.travis.yml` is so that we can collect coverage data and have it sent to Coveralls.  For local tox purposes, you can probably just omit this part.